Firestore/core/src/core/query.h

```diff
@@ -277,25 +277,36 @@ class Query {
   // sort at the end.
   std::vector<OrderBy> explicit_order_bys_;
 
-  // The memoized list of sort orders.
-  mutable std::vector<OrderBy> memoized_normalized_order_bys_;
-
   int32_t limit_ = Target::kNoLimit;
   LimitType limit_type_ = LimitType::None;
 
   absl::optional<Bound> start_at_;
   absl::optional<Bound> end_at_;
 
+  Target ToTarget(const std::vector<OrderBy>& order_bys) const;
+
+  /**
```

Comment:
nit: convert this comment to an inline comment (i.e. each line beginning with `//`). The `/** */` style of comment is conventionally used to document functions, classes, and variables.